sdcm/cluster.py

```diff
@@ -1356,6 +1356,10 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster',
         self.nemesis = []
         self.params = params
         self.add_nodes(n_nodes)
+        for node in self.nodes:
+            status = node.remoter.run('sudo systemctl status scylla-server.service', ignore_status=True)
+            if 'Active: inactive (dead)' in status.stdout:
+                self.fail("some nodes failed to start")
```

Comment:
@roydahan Yes. If we use AMI, the scylla-server service should be ready(active) after adding node. But for other backends, we might install and setup scylla later, it's normal that the service isn't ready after adding node.
 
I had revert the patch.  @abvgedeika